specification/storagetasks/resource-manager/Microsoft.StorageTasks/stable/2023-01-01/storagetasks.json

```diff
@@ -0,0 +1,903 @@
+      "description": "The response from the List Storage Tasks operation."
+    },
+    "StorageTaskAssignmentsListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StorageTaskAssignmentId"
+          },
+          "x-ms-identifiers": [],
+          "description": "Gets the list of storage task assignment Ids."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Request URL that can be used to query next page of storage task assignment Ids. Returned when total number of requested storage task assignment Ids exceed maximum page size.",
+          "readOnly": true
+        }
+      },
+      "description": "The response from the List Storage Tasks operation."
+    },
+    "StorageTaskAssignmentId": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "ARM Id of the storage task assignments, associated with the storage tasks.",
+          "readOnly": true
+        }
+      },
+      "description": "Fetch the Storage task assignment ARM ids."
+    },
+    "StorageTasksRunSummaryList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StorageTaskRunStatus"
+          },
+          "x-ms-identifiers": [],
+          "description": "Gets storage tasks run result summary."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Request URL that can be used to query next page of storage task run results summary. Returned when the number of run instances and summary reports exceed maximum page size.",
+          "readOnly": true
+        }
+      },
+      "description": "Fetch Storage Tasks Run Summary."
+    },
+    "StorageTaskRunStatus": {
+      "type": "object",
+      "properties": {
+        "taskAssignmentId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Represents the Storage Task Assignment Id associated with the storage task that provided an execution context."
+        },
+        "storageAccountId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Represents the Storage Account Id where the storage task definition was applied and executed."
+        },
+        "startTime": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Start time of the run instance. Filter options such startTime gt '2023-06-26T20:51:24.4494016Z' and other comparison operators can be used as described for DateTime properties in https://learn.microsoft.com/en-us/rest/api/storageservices/querying-tables-and-entities#supported-comparison-operators"
+        },
+        "finishTime": {
+          "readOnly": true,
+          "type": "string",
```

Comment:
>string [](http://example.com/codeflow?start=19&length=6)

Format as datetime?

Reply:
so only the UTC 8601 formats are supported from the server side. (Roundtrip, sorted, universal formatted)
making it datetime may allow customers to put in unsupported values and hence made it as string